AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -277,6 +277,7 @@ - (void)_initializeInstance
   _contentsScaleForDisplay = ASScreenScale();
   _displaySentinel = [[ASSentinel alloc] init];
   _preferredFrameSize = CGSizeZero;
+  _pendingViewState = [_ASPendingState new];
```

Comment:
OK cool, I'll load it lazily. Since we may use it often during the lifetime of a given node, do you think it's worth making the accessor a C function e.g. `ASDisplayNodeGetPendingState(ASDisplayNode *)`? Or should I use a normal Objective-C lazy accessor?